pkgs/by-name/su/sudo-rs/package.nix

```
@@ -75,7 +75,11 @@ rustPlatform.buildRustPackage (finalAttrs: {
   ];
 
   passthru = {
-    updateScript = nix-update-script { };
+    updateScript = nix-update-script {
+      extraArgs = [
+        "--version-regex=^v([0-9]+\\.[0-9]+\\.[0-9])$"
```

What most people do is the following:
```suggestion
        "--version-regex"
        "^v([0-9.]+)$"
```
Of course there is nothing wrong with forcing more specific 3-number semver.

---

Yeah, I don't have a strong opinion. You'd prefer me to change this? I found the more precise regex a bit nicer, but it's really not that important. As long as we stop getting those annoying bot PRs every week.

---

Eh, either works. I just wanted to make sure you knew about the simple version, but i will not block on this. Splitting the lines instead of `=` seems common, but i also don't see any doc about that, and no doc = no block. Restricting the update script seems fine, if you have a look and make sure it works in the future, then you can probably merge as-is.